Keas.Mvc/webpack.config.js

```diff
@@ -26,7 +26,11 @@ module.exports = env => {
           "isomorphic-fetch",
           "react",
           "react-dom",
-          "reactstrap"
+          "reactstrap",
+          "react-router-dom",
+          "react-table",
+          "react-bootstrap-typeahead",
+          "react-datepicker"
```

Comment:
@laholstege how was this working before if we weren't loading the bootstrap typeahead in our vendor js?  Were we including it from somewhere else?